Lib/difflib.py

```diff
@@ -1054,7 +1052,7 @@ def IS_LINE_JUNK(line, pat=re.compile(r"\s*(?:#\s*)?$").match):
     False
     """
 
-    return pat(line) is not None
+    return line.strip() == "" or line.lstrip().rstrip() == "#"
```

Comment:
```suggestion
    return line.strip() in ('', '#')
```

Reply:
Done, but I don't think this is sufficient.  See my review.